.github/CODEOWNERS

```diff
@@ -134,7 +134,6 @@
 
 # Test runner
 
-/test/message/test_runner_* @nodejs/test_runner
 /test/parallel/test-runner-* @nodejs/test_runner
```

Comment:
Not in this PR, but perhaps we should move all the test runner stuff into `/test/test-runner`?

Reply:
I'm +0 on that